### PR TITLE
Adjust quickstart to not show commands example

### PR DESCRIPTION
**Pull request by HyperGH**
## Summary

I thought this could be changed because new users might get the wrong impression that this is the intended
way to create commands, instead of using ext.commands. I also added a reference to ext.commands in the steps below.
P.S.: I am about 99% certain the way I did that reference is wrong, so if someone more experienced could take a look, it would
be greatly appreciated! Thanks!

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: docs/quickstart.rst [hunk 1]
```diff
@@ -32,7 +32,7 @@ It looks something like this:
         if message.author == client.user:
             return
 
-        if message.content.startswith('$hello'):
+        if message.content == ('Hello World!'):
```
Comment (Rapptz) [inline docs/quickstart.rst:35]:
```suggestion
        if message.content == 'Hello World!':
```

File: docs/quickstart.rst [hunk 2]
```diff
@@ -54,8 +54,9 @@ There's a lot going on here, so let's walk you through it step by step.
 4. Since the :func:`on_message` event triggers for *every* message received, we have to make
    sure that we ignore messages from ourselves. We do this by checking if the :attr:`Message.author`
    is the same as the :attr:`Client.user`.
-5. Afterwards, we check if the :class:`Message.content` starts with ``'$hello'``. If it is,
-   then we send a message in the channel it was used in with ``'Hello!'``.
+5. Afterwards, we check if the :class:`Message.content` is ``'Hello World!'``. If it is,
+   then we send a message in the channel it was used in with ``'Hello!'``. This is a basic way of 
+   handling commands, which can be later automated with the :ref:`ext.commands` framework.
```
Comment (Rapptz) [inline docs/quickstart.rst:59]:
```suggestion
5. Afterwards, we check if the :class:`Message.content` is ``'Hello World!'``. If it is,
   then we send a message in the channel it was used in with ``'Hello!'``. 
```

**Review by Rapptz — CHANGES_REQUESTED** [review]

**Comment by HyperGH** [comment]
Updated the PR based on feedback on Discord

File: docs/quickstart.rst (hunk 2 above)
Comment (HyperGH) [reply docs/quickstart.rst]: I strongly think that a reference to ext.commands should be included somewhere on this page, if not here then somewhere else.

File: docs/quickstart.rst [hunk 3]
```diff
@@ -32,7 +32,7 @@ It looks something like this:
         if message.author == client.user:
             return
 
-        if message.content.startswith('$hello'):
+        if message.content.startswith('hello'):
```
Comment (Rapptz) [inline docs/quickstart.rst:35]:
```suggestion
        if message.content.startswith('$hello'):
```

File: docs/quickstart.rst [hunk 4]
```diff
@@ -54,8 +54,9 @@ There's a lot going on here, so let's walk you through it step by step.
 4. Since the :func:`on_message` event triggers for *every* message received, we have to make
    sure that we ignore messages from ourselves. We do this by checking if the :attr:`Message.author`
    is the same as the :attr:`Client.user`.
-5. Afterwards, we check if the :class:`Message.content` starts with ``'$hello'``. If it is,
-   then we send a message in the channel it was used in with ``'Hello!'``.
+5. Afterwards, we check if the :class:`Message.content` starts with ``'hello'``. If it does,
```
Comment (Rapptz) [inline docs/quickstart.rst:57]:
```suggestion
5. Afterwards, we check if the :class:`Message.content` starts with ``'$hello'``. If it does,
```

**Review by Rapptz — CHANGES_REQUESTED** [review]

**Review by Rapptz — APPROVED** [review]

**Pull request merged**